### ISSUE TITLE
[Documentation]: function docstrings not rendering properly for `pynwb.file module`

### ISSUE THREAD

**Issue by h-mayorquin** [NeurodataWithoutBorders/pynwb#1444]
### What would you like changed or added to the documentation and why?

There seems to be some problem as the docstrings are not rendered properly:
https://pynwb.readthedocs.io/en/stable/pynwb.file.html#pynwb.file.NWBFile.add_unit_column

The documentation seems to break at the description of index:

![Untitled drawing (1)](https://user-images.githubusercontent.com/6429509/159972812-7cd3cc0d-4c85-4625-9613-e1bfb4efbce6.png)

### Do you have any interest in helping write or edit the documentation?

No.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/NeurodataWithoutBorders/pynwb/blob/dev/.github/CODE_OF_CONDUCT.rst)
- [X] Have you checked the [Contributing](https://github.com/NeurodataWithoutBorders/pynwb/blob/dev/docs/CONTRIBUTING.rst) document?
- [X] Have you ensured this change was not already [requested](https://github.com/NeurodataWithoutBorders/pynwb/issues)?

**Comment by oruebel** [comment]
Looking at code, I believe this is actually a problem inherited from HDMF ``DynamicaTable.add_column``. I'll open a corresponding issue on HDMF and close this one in favor of that one.

**Comment by oruebel** [comment]
I've created a new issue for this on the HDMF repo since this issue needs to be fixed in HDMF. The new issue is here https://github.com/hdmf-dev/hdmf/issues/711 I'm closing this issue for now as this will get fixed automatically once that issue is complete.
